bool use_latest
string trajectory_filename
---
bool success
string message
